zgw/object-management/src/main/kotlin/com/ritense/objectmanagement/domain/search/SearchRequestValue.kt

```diff
@@ -0,0 +1,6 @@
+package com.ritense.objectmanagement.domain.search
+
+class SearchRequestValue(
```

Comment:
I get an error here:
```
JSON parse error: Cannot construct instance of `com.ritense.objectmanagement.domain.search.SearchRequestValue`
```

Reply:
Seems I has to add something to make it work on my mac. But that might just be me